client/src/components/AddList.js

```diff
@@ -0,0 +1,231 @@
+    }
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        if (validations()) {
+            fetch("route", {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                    'title': title,
+                    //'cover': cover
+                })})
+                .then(response => response.json())
+                .then(function(response) {
+                    if (response.status === 'success') {
+                        console.log('Success:');
+                    }
+                    else {
+                        window.alert(response.message); // Replace with snackbar.
+                        console.log(response.message);
+                    }
+                })
+                .catch((error) => {
+                    console.error('Error:', error);
+                })
+            }
+    }
+
+    return (
+        <Modal
+        className={classes.popup}
+        open={addListOpen}
+        onClose={handleClose}
+        aria-labelledby="add-list"
+        aria-describedby="confirms-adding-a-new-list"
+        >
+        <Box className={classes.paper}>
+          <Box className={classes.closeButtonContainer}>
+            <IconButton
+              aria-label="close"
+              className={classes.closeButton}
+              onClick={handleClose}
+              size="small"
+            >
+              <CloseIcon fontSize="small" />
+            </IconButton>
+          </Box>
+          <Box className={classes.titleContainer}>
+            <Typography><h2 className={classes.titleText}>Create new list</h2></Typography>
+          </Box>
+          <Box className={classes.bodyContainer}>
+            <Typography><h4 className={classes.itemText}>Add a title</h4></Typography>
+          </Box>
+          <Box className={classes.textFieldContainer}>
+            <TextField
+              className={classes.textField}
+              variant="outlined"
+              placeholder="Enter name" // Placeholder needs to be centered.
+              fullWidth
+              type="text"
+              error={!!errors.title}
+              helperText={errors.title}
+              onChange={(e)=>setTitle(e.target.value)}/>
+          </Box>
+          <Box className={classes.bodyContainer}>
+            <Typography><h4 className={classes.itemText}>Add a cover</h4></Typography>
+          </Box>
+          <Box className={classes.imageFieldContainer}>
+            <Typography>Drop an image here or <strong><u><input
+              accept="image/*"
+              className={classes.input}
+              id="file-input"
+              multiple
+              type="file"
+              />
+            <label htmlFor="file-input">
+              select a file
+            </label></u></strong></Typography>
+          </Box>
```

Comment:
For the text of `<imageFieldContainer />`, I would structure it the following way:

```
<Box className={classes.imageFieldContainer}>
  <Typography>Drop an image here or </Typography>
  <Typography>
    <input ... />
    <label ... >select a file</label>
  </Typography>
</Box>
```

That way when you style `imageFieldContainer` to use `flexDirection: "column"`, the text will both sit on separate lines.

Also add a class to `<label>` and use css to make it bold and underlined instead of using HTML tags. The `<u>` and `<strong>` tags are usually considered obsolete.